pretyping/vnorm.ml

```diff
@@ -352,7 +373,7 @@ and nf_cofix env sigma cf =
   mkCoFix (init,(name,cft,cfb))
 
 let cbv_vm env sigma c t  =
-  if Termops.occur_meta_or_existential sigma c then
+  if Termops.occur_meta sigma c then
     CErrors.user_err Pp.(str "vm_compute does not support existential variables.");
```

Comment:
Please change "existential variables" to "metavariables" in the message.

Reply:
...this reminds me that the reference manual would probably be clearer if it could make up its mind between the terminology metavariable vs existential variables (or at least once for all say somewhere that they are essentially synonym).

Reply:
Hum, I spoke too quickly. The reference manual is actually ok (except in the ssr chapter which uses the cute terminology of "existential metavariables"). Otherwise, the terminology is stable:
- metavariables = Ltac pattern-matching variables
- existential variables = holes in terms